src/js/components/Clock/doc.js

```diff
@@ -166,4 +166,64 @@ export const themeDoc = {
     type: 'string',
     defaultValue: '288px',
   },
+  'clock.digital.text.xsmall.size': {
+    description: 'Defines the font size of the Digital Clock',
+    type: 'string',
+    defaultValue: '10px',
+  },
+  'clock.digital.text.xsmall.height': {
+    description: 'Defines the line height of the Digital Clock',
+    type: 'string',
+    defaultValue: '1.5',
```

Comment:
I think a more accurate definition for height is the type of 'number' rather than 'string'.
An example of that will be the test you created for the use case of customSize with using height as a number and not as a string.
(This comment is relevant for all height values on different sizes)

Reply:
Sure! Didn't notice that. Will fix them all :smile: